servers_v6.json

```
@@ -19,6 +19,10 @@
     "name": "C.A.M.S.",
     "address": ["routerchain.ddns.net", "nikochio.ddns.net"]
   },
+  {
+    "name": "mindustry.ru",
+    "address": ["mindustry.ru"],
```

Can we just remove the comma? It's just that bit annoying...
```suggestion
    "address": ["mindustry.ru"]
```